packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -607,7 +645,7 @@ export class AngularCompilerPlugin implements Tapable {
   // It skips the program creation because we need to use `loadNgStructureAsync()`,
   // and uses CustomTransformers.
   // Should be Program and CustomTransformers from @angular/compiler-cli instead of any.
-  private _emit(program: any, customTransformers: any) {
+  private _emit(program: ts.Program & any, customTransformers: ts.CustomTransformers & any) {
```

Comment:
It seems to me those `& any` are wrong. You probably want a `declare type Program = ...` and `declare type CustomTransformers = ...` at the top of the file to remove those weird typings in arguments.